repos/system_upgrade/common/actors/systemd/checksystemdbrokensymlinks/libraries/checksystemdbrokensymlinks.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+from leapp import reporting
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SystemdBrokenSymlinksSource
+
+FMT_LIST_SEPARATOR = '\n - '
+
+
+# TODO wording
+def _report_broken_symlinks(symlinks):
+    summary = (
+        'Leapp detected broken systemd symlinks on the system.'
+        'The symlinks will not be handled during the upgrade.'
+        'The found broken symlinks: {}'
+    )
+
+    hint = (
+        'You can use systemctl disable to remove these symlinks'
+    )
+
+    reporting.create_report([
+        reporting.Title(
+            'Broken systemd symlinks found on the system'
+        ),
+        reporting.Summary(summary.format(_printable_symlinks(symlinks))),
+        reporting.Remediation(hint=hint),
```

Comment:
Wondering if it does make sense to provide also a remediation command as we should know all the information.
@pirat89 WDYT? ^

Reply:
yes. In this particular case, `rm <files>` could be good.